recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+
+build:
+  number: 0
+  features:
+    - vc14    [win]
+
+requirements:
+  build:
+    - m4 [unix]
```

Comment:
Add at least two spaces between the selector and package. Also use the comment style as mentioned above.